x-pack/plugins/search_playground/public/components/sources_panel/create_index_callout.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButton, EuiCallOut, EuiSpacer, EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React from 'react';
+import { useChatContext } from '../../hooks/use_chat_context';
+
+export const CreateIndexCallout: React.FC = () => {
+  const { navigateToIndexPage } = useChatContext();
+
+  return (
+    <EuiCallOut
+      title={i18n.translate('xpack.searchPlayground.sources.createIndexCallout.headerText', {
+        defaultMessage: 'Create an index',
+      })}
+      color="primary"
+      iconType="iInCircle"
+    >
+      <EuiText size="s">
+        <p>
+          <FormattedMessage
+            id="xpack.searchPlayground.sources.createIndexCallout.description"
+            defaultMessage="You can not continue without having at least one index with data to search."
+          />
+        </p>
+      </EuiText>
+      <EuiSpacer size="l" />
+      <EuiButton
```

Comment:
This is where we would use the locator.

One thing that came up in the frontend alignment meeting when we discussed this is that there is not a great place to send users in Serverless currently. The index management page has a "create index" button, but it opens a simple modal and doesn't have route or anything. We may end up need to make this optional or sending the user to the Getting Started page in Serverless. Which is still more reason to use a Locator. If the locator is not found we'd omit this button.